include/openthread/cli.h

```diff
@@ -32,12 +32,12 @@
  *  This file defines the top-level functions for the OpenThread CLI server.
  */
 
-#ifndef CLI_CONSOLE_H_
-#define CLI_CONSOLE_H_
+#ifndef CLI_H_
+#define CLI_H_
 
 #include <stdint.h>
 
-#include <openthread-types.h>
+#include "openthread/types.h"
```

Comment:
Overall looks good.  Curious as to why "" is used for openthread library includes of instead <>?